opsdroid/tests/test_web.py

```diff
@@ -0,0 +1,103 @@
+"""Test the opsdroid web."""
+import ssl
+
+import asynctest.mock as amock
+
+from opsdroid.cli.start import configure_lang
+from opsdroid import web
+import aiohttp.web
+
+configure_lang({})
+
+
+async def test_web(opsdroid):
+    """Create a web object and check the config."""
+    app = web.Web(opsdroid)
+    assert app.config == {}
+
+
+async def test_web_get_port(opsdroid):
+    """Check the port getter."""
+    opsdroid.config["web"] = {}
+    app = web.Web(opsdroid)
+    assert app.get_port == 8080
+
+    opsdroid.config["web"] = {"port": 8000}
+    app = web.Web(opsdroid)
+    assert app.get_port == 8000
+
+
+async def test_web_get_host(opsdroid):
+    """Check the host getter."""
+    opsdroid.config["web"] = {}
+    app = web.Web(opsdroid)
+    assert app.get_host == "0.0.0.0"
+
+    opsdroid.config["web"] = {"host": "127.0.0.1"}
+    app = web.Web(opsdroid)
+    assert app.get_host == "127.0.0.1"
+
+
+async def test_web_get_ssl(opsdroid):
+    """Check the host getter."""
+    opsdroid.config["web"] = {}
+    app = web.Web(opsdroid)
+    assert app.get_ssl_context is None
+
+    opsdroid.config["web"] = {
+        "ssl": {"cert": "tests/ssl/cert.pem", "key": "tests/ssl/key.pem"}
+    }
+    app = web.Web(opsdroid)
+    assert isinstance(app.get_ssl_context, type(ssl.SSLContext(ssl.PROTOCOL_SSLv23)))
+    assert app.get_port == 8443
+
+    opsdroid.config["web"] = {
+        "ssl": {"cert": "/path/to/nonexistant/cert", "key": "/path/to/nonexistant/key",}
```

Comment:
I can understand why the whole conflict is happening. Flake expects a space because it sees that comma inside the dict so it's probably assuming that a new key should be there.

I'd say that the best way to deal with this, is to just remove that comma altogether since it's not doing anything there really 👍 

```suggestion
        "ssl": {"cert": "/path/to/nonexistant/cert", "key": "/path/to/nonexistant/key"}
```